### ISSUE TITLE
Using JFXColorPicker with javafxports on android(with fix)

### ISSUE THREAD

**Issue by Sashie** [jfoenixadmin/JFoenix#306]
Changed in `JFXColorPickerUI` due to streams not being compatible...
```java
	private void setColorAtLocation(int x, int y) {
		if(allowColorChange){
			Color color = getColorAtLocation(x, y);
			String colorString = "rgb("+color.getRed()*255+"," +color.getGreen()*255+ "," +color.getBlue()*255+ ");";
			colorNodes.forEach(node-> node.setStyle("-fx-background-color:" + colorString + "; -fx-fill:" + colorString+";"));
		}
	}
```
to
```java
	private void setColorAtLocation(int x, int y) {
		if(allowColorChange){
			Color color = getColorAtLocation(x, y);
			String colorString = "rgb("+color.getRed()*255+"," +color.getGreen()*255+ "," +color.getBlue()*255+ ");";
			for (Node node : colorNodes)
				node.setStyle("-fx-background-color:" + colorString + "; -fx-fill:" + colorString+";");
		}
	}
```
(tested and works)

**Comment by jfoenixadmin** [comment]
Hello,
Thanks for the fix 👍  can you create a pull request to merge the fix with master branch?

Regards,

**Comment by jfoenixadmin** [comment]
Will fix it without a pull request in order to include the fix in JFoenix 1.4

Regards,

**Comment by Sashie** [comment]
perhaps by then i'll have learned how to do a proper pull request ^.^
